requester/requester_internal_test.go

```diff
@@ -29,7 +32,9 @@ func TestRun(t *testing.T) {
 			label: "return ErrRequest early on request error",
 			req: New(config.Config{
 				Request: config.Request{
+					Method:  "POST",
 					Timeout: 0,
+					Body:    validBody,
```

Comment:
Is it necessary to add `Method` and `Body` in all these tests?

Reply:
I guess that no :sweat_smile: 
I have just let it in the happy path to have a full one.